locations/ak36.yml

```diff
@@ -145,3 +145,7 @@ gre_base_prefix: 2001:bf7:750:4003::/64
 gre_tunnel_alias: ak36
 
 wireless_profile: disable
+
+location__ssh_keys__to_merge:
+  - comment: rtznprmpftl
+    key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHZ5H1Y3o30QvU3ohtXiWZUgw9fIGxvW35STG7vICF8O localghost
```

Comment:
I suggest to add the key of rtznprmpftl to the default list of trusted keys instead.